.buildkite/Dockerfile

```diff
@@ -15,4 +15,3 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
 RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
 RUN apt-get -y update
 RUN apt-get -y install yarn
-RUN rustup component add clippy --toolchain nightly-2020-02-05-x86_64-unknown-linux-gnu
```

Comment:
Nope, it's just part of the base image.

Reply:
And does this registry base image have the same rust-toolchain version the coco dependency? Will both dependencies always be on the same version?

Reply:
We currently don't actively coordinate the toolchain versions across projects. Historically registry was the first to adopt nightly, so we have been trailing theirs. They also off the base image conveniently. This could be raised in radicle-decisions as a cross-engineering concern.